lib/microformats2/results/collection.rb

```diff
@@ -0,0 +1,114 @@
+module Microformats2
+
+  #stub to get around the tests for now
+  class Collection
+    def initialize(hash)
+      @hash = hash
+    end
+    def to_hash
+      @hash
+    end
```

Comment:
This is fine, but for future resilience and DRYer code, you should use `alias_method`.

`alias_method :baz, :foo`